api/tests/core/search/test_api.py

```diff
@@ -128,9 +129,13 @@ def test_index_new_venue(self):
     def test_unindex_ineligible_venues(self):
         venue1 = offers_factories.VenueFactory(isPermanent=False)
```

Comment:
(détail) On pourrait ajouter un lien indexable, qui nous servirait d'indicateur pour confirmer qu'on désindexe les liens inéligibles, certes, mais qu'on indexe (garde) bien le lieu éligible. Normalement c'est assuré par le test précédent, mais s'il y a un bug dans ce test-ci, il pourrait passer inaperçu.